### DISCUSSION TITLE
Something New is Cooking!

### DISCUSSION THREAD

**Discussion by Shabinder** [Shabinder/SpotiFlyer#58] [Show and tell]
All **New Feature Updates** will be shared here!

**Comment by Shabinder** [comment]
https://user-images.githubusercontent.com/44765050/109874405-189f5a80-7c95-11eb-83f6-667f26738f66.mp4

**SNEAK PEEK...**

_STAY TUNED_

**Reply by ghost** [reply]
Thanks for great app.
